lib/uktt/geographical_area.rb

```diff
@@ -0,0 +1,41 @@
+module Uktt
+  class GeographicalArea
+    attr_accessor :config, :response
+
+    def initialize(opts = {})
+      Uktt.configure(opts)
+      @config = Uktt.config
+      @response = nil
+    end
+
+    def retrieve
+      fetch "#{GEOGRAPHICAL_AREAS}"
+    end
+
+    def config=(new_opts = {})
+      merged_opts = Uktt.config.merge(new_opts)
+      Uktt.configure merged_opts
+      @config = Uktt.config
+    end
+
+    def find_in(arr)
+      return '@response is nil, run #retrieve first' unless @response
+  
+      response = @response.included.select do |obj|
+        arr.include?(obj.id)
+      end
+      response.length == 1 ? response.first : response
```

Comment:
So if response = [1], you return 1, but if it is [1,2,3,4] you return [1,2,3,4]... Hmmm, can we not just return an array all the time? E.g [1] or [1,2,3,4]

Reply:
I actually don't make use of this method, personally. I rely on the json api deserializer to give me an easy format

Reply:
Probably we can remove it